src/extension/src/ProcessHandler.py

```diff
@@ -55,20 +58,124 @@ def get_env_settings(self, ext_env_handler):
         return env_settings
 
     def start_daemon(self, seq_no, config_settings, ext_env_handler):
-        """ Launches the core code in a separate independent process with required arguements and exits the current process immediately """
+        """ Launches the core code in a separate independent process with required arguments and exits the current process immediately """
         exec_path = os.path.join(os.getcwd(), Constants.CORE_CODE_FILE_NAME)
         public_config_settings = base64.b64encode(json.dumps(self.get_public_config_settings(config_settings)).encode("utf-8")).decode("utf-8")
         env_settings = base64.b64encode(json.dumps(self.get_env_settings(ext_env_handler)).encode("utf-8")).decode("utf-8")
 
         args = " -sequenceNumber {0} -environmentSettings \'{1}\' -configSettings \'{2}\'".format(str(seq_no), env_settings, public_config_settings)
-        command = ["python " + exec_path + " " + args]
+
+        # Verify the python version available on the machine to use
+        self.logger.log("Python version: " + " ".join(sys.version.splitlines()))
+        python_cmd = self.get_python_cmd()
+        if python_cmd == Constants.PYTHON_NOT_FOUND:
+            self.logger.log("Cannot execute patch operation due to [Error={0}]".format(Constants.PYTHON_NOT_FOUND))
+            return
+
+        command = [python_cmd + " " + exec_path + " " + args]
         self.logger.log("Launching process. [command={0}]".format(str(command)))
         process = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
         if process.pid is not None:
             self.logger.log("New shell process launched successfully. [Process ID (PID)={0}]".format(str(process.pid)))
+            # Wait for 5 seconds
+            time.sleep(5)
+            # if process is not running, log stdout and stderr
+            if process.poll() is not None:
+                self.logger.log("Process not running for [sequence={0}]".format(seq_no))
+                self.logger.log("Standard output for the inactive process: [Output={0}]".format(str(process.stdout)))
+                self.logger.log("Error logs for the inactive process: [Error={0}]".format(str(process.stderr)))
```

Comment:
Minor:
Stderr for the inactive process since it's an industry standard term. For consistency, Stdout for line 85.
If more friendly terms are preferred, 'Output' instead of 'Standard output' in 85 and 'Errors reported for_[/by]_.. '.
Whichever you prefer but it's preferrable to stay consistent.